src/main/js/isd.js

```diff
@@ -246,7 +246,7 @@
 
             if (!(cs.qname in spec_attr)) continue;
 
-            if (cs.compute !== null) {
+            if (cs.compute !== undefined && cs.compute !== null) {
```

Comment:
This should not be needed since an undefined `compute` will not yield an exception